APP_LINK_SETUP.md

```diff
@@ -0,0 +1,34 @@
+## Set Up App Links
+
+In order to use the PayPal App Switch flow your application must be set up for [App Links](https://developer.android.com/training/app-links?authuser=2).
```

Comment:
Should this just say PayPal flow vs App Switch?

Reply:
Ah yes good catch - updated